### DISCUSSION TITLE
[Feature Request] Alert/Callout Component (As shown in docs)

### DISCUSSION THREAD

**Discussion by douglasg14b** [quasarframework/quasar#15557] [Ideas / Proposals] [answered]
The docs have a callout rendered in many places, but this does not seem to exist in Quasar. Given it's used in the docs, it seems more likely that it's a valuable nice to have.

Similar to the vuetify `Alert` ([link](https://vuetifyjs.com/en/components/alerts/)) a `Callout` would be very nice to have. 

Example from the quasar docs:

![image](https://user-images.githubusercontent.com/1400380/224891688-c2774050-fd36-4c5c-b049-b07a03306b57.png)

**Comment by MilosPaunovic** [comment, marked as answer]
It is just a styled `div`. You can inspect it in the docs and create an exact copy in a couple of minutes.
